libs/androidlib/src/mill/androidlib/AndroidR8AppModule.scala

```diff
@@ -60,19 +59,15 @@ trait AndroidR8AppModule extends AndroidAppModule {
       .flatMap(_.proguardRules)
   }
 
+  /**
+   * The ProGuard/R8 rules configuration files for the Android project.
+   * @return
+   */
+  def androidProjectProguardFiles: T[Seq[PathRef]] = Task.Sources()
```

Comment:
Is "proguard file" the default idiom used for these config files? If there are more specific terms like "proguard rules" or "proguard configs", we probably should use these.

Reply:
I like the `androidProjectProguardRules` better :+1:

Reply:
in gradle though they are referred as
`proguardFiles(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro")`